certification/internal/shell/deployable_by_olm.go

```diff
@@ -0,0 +1,63 @@
+package shell
```

Comment:
So... This probably shouldn't go in shell? It's using an actual golang client, as opposed to shell commands. I don't really have a suggested package name. @komish thoughts?

Reply:
I liked `client`. which I think @samira-barouti already defined. 

With that said, we're getting dangerously close to blurring the line between our proposed "v1" and "v2" implementations, where here "v1" meant exec-ing out to shell commands, and "v2" meant explicitly doing everything using as close to a pure-go mentality as we could. 

Do we think we need to adapt our internal structure a bit to fit this direction?